etna/datasets/tsdataset.py

```diff
@@ -679,6 +679,25 @@ def regressors(self) -> List[str]:
         """
         return self._regressors
 
+    # TODO: is it ok that it includes target components and prediction intervals?
+    # TODO: don't we want to rename it into "variables" or smth like this for example?
+    # TODO: what about adding this into ts.describe/ts.info?
```

Comment:
This needs a discussion

Reply:
Can you elaborate on this one?

Reply:
Look at the contents of `ts.info` in eda notebook. Don't we want to add number of features into this table.

Reply:
Decided to exclude components/intervals from "num_exogs" in `ts.info`.